### PR TITLE
Aws installation

### PR DESCRIPTION
**Pull request by ru-fu**

### CODE REVIEW

File: install-appliance.md [hunk 1]
```diff
@@ -3,25 +3,114 @@ This offering is well suited for initial prototype and small scale deployments.
...
+
+Installing the Anbox Cloud Appliance through the AWS Marketplace simplifies the installation and deployment process and allows billing to be handled directly through AWS. The following steps guide you through all relevant steps to deploy the Anbox Cloud Appliance in your AWS account.
+
+#### Choose an instance architecture
+
+First, decide if you want to deploy the Anbox Cloud Appliance on an x86 or Arm instance architecture. AWS supports both.
+
+The decision should factor in the following aspects:
+
+* GPUs are currently available for x86. Nvidia GPUs will only become available for Arm instances [later in 2021](https://aws.amazon.com/blogs/machine-learning/aws-and-nvidia-to-bring-arm-based-instances-with-gpus-to-the-cloud/).
```
Comment (morphis) [inline install-appliance.md:39]:
```suggestion
* GPUs are currently available for x86. NVIDIA GPUs will only become available for Arm instances [later in 2021](https://aws.amazon.com/blogs/machine-learning/aws-and-nvidia-to-bring-arm-based-instances-with-gpus-to-the-cloud/).
```

File: install-appliance.md [hunk 2]
```diff
@@ -3,25 +3,114 @@ This offering is well suited for initial prototype and small scale deployments.
...
+
+#### Choose an instance type
+
+AWS offers various instance types. The Anbox Cloud Appliance images are only listed for a subset of the available instance types.
+
+Select the instance type that is most suitable for what you're planning to do. For example, if you just want to try out the Anbox Cloud Appliance, an instance type with GPU support and limited CPU and memory is sufficient.
+
+![00_appiance-aws-instance-type|690x532](upload://AskBufPuBZg586bSaI6Eg39XGqy.png) 
+
+In this example, we picked *g4dn.xlarge*, which provides 4 vCPUs, 16 GB of memory and a single Nvidia Tesla T4 GPU.
```
Comment (morphis) [inline install-appliance.md:57]:
```suggestion
In this example, we picked *g4dn.xlarge*, which provides 4 vCPUs, 16 GB of memory and a single NVIDIA Tesla T4 
GPU.
```

**Review by morphis — APPROVED** [review]
Looks great!

Just left two small suggestions, otherwise ready to go

**Comment by ru-fu** [comment]
Thanks! Updated.

**Pull request merged**